sys/kern/kern_tc.c

```diff
@@ -220,7 +220,7 @@ bintime_off(struct bintime *bt, u_int off)
 	do {
 		th = timehands;
 		gen = atomic_load_acq_int(&th->th_generation);
-		btp = (struct bintime *)((vm_offset_t)th + off);
+		btp = (struct bintime *)((uintptr_t)th + off);
```

Comment:
`char *` would be better tbh, `uintptr_t` for normal pointer arithmetic is bad practice